content/shmem_put_signal.tex

```diff
@@ -0,0 +1,83 @@
+\color{Green}
+\apisummary{
+    The put with signal routines provide a method for copying data from a
+    contiguous local data object to a data object on a specified \ac{PE}
+    and set a remote flag to signal completion.
```

Comment:
Suggest:
* "put with signal" &#8594; "put-with-signal"
* "...and set a remote flag..." &#8594; "...and subsequently setting a remote flag..."

Reply:
resolved - closing outdated changes.